content/influxdb/v2.0/process-data/manage-tasks/create-task.md

```diff
@@ -120,3 +121,17 @@ option task = {
 
 # <ctrl-d> to close the pipe and submit the command
 ```
+
+## Create a task using the InfluxDB API
+
+```
+curl --location --request POST 'https://us-west-2-1.aws.cloud2.influxdata.com/api/v2/tasks' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Token <YOURTOKEN>' \
+--data-raw '{
+    "flux": "option task = {name: \"CPU Total 1 Hour New\", every: 1h}\n\nfrom(bucket: \"telegraf\")\n\t|> range(start: -1h)\n\t|> filter(fn: (r) =>\n\t\t(r._measurement == \"cpu\"))\n\t|> filter(fn: (r) =>\n\t\t(r._field == \"usage_system\"))\n\t|> filter(fn: (r) =>\n\t\t(r.cpu == \"cpu-total\"))\n\t|> aggregateWindow(every: 1h, fn: max)\n\t|> to(bucket: \"cpu_usage_user_total_1h\", org: \"russ+cloud2'\''s organization\")",
+    "orgID": "<YOURORGID>",
+    "status": "active",
+    "description": "This task downsamples CPU data every hour"
+}'
+```
```

Comment:
- Specify syntax highlighting on the code block.
- For simplicity, we can probably leave out the `--location` flag. This just tells the request to follow redirects.
- Totally nit-picky, so feel free to ignore, but I like to indent lines after an escaped newline in commands to show that it's still part of the same command:
- 
~~~suggestion
```sh
curl --request POST 'https://us-west-2-1.aws.cloud2.influxdata.com/api/v2/tasks' \
  --header 'Content-Type: application/json' \
  --header 'Authorization: Token <YOURTOKEN>' \
  --data-raw '{
    "flux": "option task = {name: \"CPU Total 1 Hour New\", every: 1h}\n\nfrom(bucket: \"telegraf\")\n\t|> range(start: -1h)\n\t|> filter(fn: (r) =>\n\t\t(r._measurement == \"cpu\"))\n\t|> filter(fn: (r) =>\n\t\t(r._field == \"usage_system\"))\n\t|> filter(fn: (r) =>\n\t\t(r.cpu == \"cpu-total\"))\n\t|> aggregateWindow(every: 1h, fn: max)\n\t|> to(bucket: \"cpu_usage_user_total_1h\", org: \"russ+cloud2'\''s organization\")",
    "orgID": "<YOURORGID>",
    "status": "active",
    "description": "This task downsamples CPU data every hour"
}'
```
~~~